titan/population_io.py

```diff
@@ -210,15 +216,15 @@ def create_agent(
         if attr not in init_attrs:
             setattr(agent, attr, eval(val))
 
-    for feature in agent_feats:
-        feat_row = agent_feats[feature][agent.id]
-        agent_feat = getattr(agent, feature)
-        for attr, val in feat_row.items():
+    for extra in agent_extras:
```

Comment:
What is an agent_extra?

Reply:
features/exposures we staple on to agent